pkg/prometheus/promcfg_test.go

```diff
@@ -9776,6 +9776,89 @@ func TestScrapeConfigSpecConfigWithLightSailSD(t *testing.T) {
 		})
 	}
 }
+
+func TestScrapeConfigSpecConfigWithOVHCloudSD(t *testing.T) {
```

Comment:
(nit) could you also add the `Service` field to the test case?

Reply:
Got it. Done.